config/bootstrap.php

```diff
@@ -43,27 +42,6 @@
     ]);
 }
 
-Router::plugin('DebugKit', function ($routes) {
```

Comment:
I'm splitting functionality up into plugins. These plugins check for a sub domain, and create new routes based on that domain and allow certain access to the app based on the sub domain. I tried just overwriting the scope but that didn't work.

Perhaps this isn't the best approach and I'm sure my use case isn't a popular one. I didn't have anyone to consult with :P